src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1386,19 +1810,43 @@ class ongoingTasksStats extends viewModelBase {
                     }
                         break;
                     case "Raven":
-                    case "Sql":
+                    case "Sql": {
                         const elementWithData = context.rootStats as EtlPerformanceBaseWithCache;
                         if (elementWithData.BatchCompleteReason) {
                             tooltipHtml += `Batch complete reason: ${elementWithData.BatchCompleteReason}<br/>`;
                         }
-                        
+
                         if (elementWithData.CurrentlyAllocated && elementWithData.CurrentlyAllocated.SizeInBytes) {
                             tooltipHtml += `Currently allocated: ${generalUtils.formatBytesToSize(elementWithData.CurrentlyAllocated.SizeInBytes)} <br/>`;
                         }
 
                         if (elementWithData.BatchSize && elementWithData.BatchSize.SizeInBytes) {
                             tooltipHtml += `Batch size: ${generalUtils.formatBytesToSize(elementWithData.BatchSize.SizeInBytes)} <br/>`;
                         }
+                    }
+                        break;
+                    case "SubscriptionBatch": {
+                        const elementWithData = context.rootStats as SubscriptionBatchPerformanceStatsWithCache;
+                       
+                        tooltipHtml += `Document sent in batch: ${elementWithData.DocumentsCount} <br/>`;
+                        tooltipHtml += `Document size: ${elementWithData.DocumentsSize} <br/>`;
+                        tooltipHtml += `Included documents: ${elementWithData.NumberOfIncludedDocuments} <br/>`;
+                        tooltipHtml += `Included counters: ${elementWithData.NumberOfIncludedCounters} <br/>`;
+                        tooltipHtml += `Included timeseries entries: ${elementWithData.NumberOfIncludedTimeSeriesEntries} <br/>`;
+                        //tooltipHtml += `Batch Id: ${elementWithData.BatchId} <br/>`;
+
+                        if (elementWithData.Exception) {
+                            tooltipHtml += `<br>Message: ${elementWithData.Exception} <br/>`;
+                        }
+                    }
+                        break;
+                    case "AggregatedBatchesInfo": {
+                        const elementWithData = context.rootStats as SubscriptionBatchPerformanceStatsWithCache;
+
+                        tooltipHtml += `Number of batches sent: ${elementWithData.AggregatedBatchesCount } <br/>`;
+                        tooltipHtml += `Connection Id: ${elementWithData.ConnectionId } <br/>`;
+                        tooltipHtml += `Batch Id: ${elementWithData.BatchId} <br/>`;
```

Comment:
do we want to display batch id?

Reply:
more over this is aggregated item so it doesn't have batch id.

Reply:
I inserted batch id = 0 for this artificial item, 
either way, I'm removing these 2 lines as it was just for the testing